stubs/cffi/_cffi_backend.pyi

```diff
@@ -0,0 +1,286 @@
+import sys
+import types
+from _typeshed import Incomplete, ReadableBuffer, WriteableBuffer
+from collections.abc import Callable, Hashable
+from typing import Any, ClassVar, Protocol, TypeVar, overload
+from typing_extensions import Literal, TypeAlias, final
+
+_T = TypeVar("_T")
+
+class _Allocator(Protocol):
+    def __call__(self, cdecl: str | CType, init: Any = ...) -> _CDataBase: ...
+
+FFI_CDECL: int
+FFI_DEFAULT_ABI: int
+RTLD_DEEPBIND: int
+RTLD_GLOBAL: int
+RTLD_LAZY: int
+RTLD_LOCAL: int
+RTLD_NODELETE: int
+RTLD_NOLOAD: int
+RTLD_NOW: int
```

Comment:
Some of these look like they're platform-specific. On my Windows machine:

```pycon
>>> import _cffi_backend as c
>>> c.FFI_CDECL
1
>>> c.FFI_DEFAULT_ABI
1
>>> c.RTLD_DEEPBIND
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module '_cffi_backend' has no attribute 'RTLD_DEEPBIND'
>>> c.RTLD_GLOBAL
0
>>> c.RTLD_LAZY
0
>>> c.RTLD_LOCAL
0
>>> c.RTLD_NODELETE
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module '_cffi_backend' has no attribute 'RTLD_NODELETE'
>>> c.RTLD_NOLOAD
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module '_cffi_backend' has no attribute 'RTLD_NOLOAD'. Did you mean: 'RTLD_GLOBAL'?
>>> c.RTLD_NOW
0
```

Reply:
I just checked the source of the C module and there it's defined with a preprocessor macro like this:
```
#ifdef RTLD_DEEPBIND
    { "RTLD_DEEPBIND", RTLD_DEEPBIND },
#endif
```
I'll need to research which operating systems expose which flags in order to correctly map this to `if sys.platform` conditions.

Reply:
We could put the ones that aren't on my Windows machine behind `if sys.platform != "win32"` guards for now, since we can probably be confident that they don't exist on Windows. Or we can leave it to a future PR. Up to you; I'm easy either way :)